inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/http/MessageFilter.java

```diff
@@ -90,8 +90,7 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
             } else if (body.length() > maxMsgLength) {
                 LOG.warn("Received bad request from client. Body length is " + body.length());
                 code = StatusCode.EXCEED_LEN;
-                message = "Bad request from client. " + "Body length is " + body.length()
-                        + ",exceeding the limit:" + maxMsgLength;
+                message = "Bad request from client. " + "Body length is exceeding the limit:" + maxMsgLength;
```

Comment:
Not need use `+` to append the two strings.

Reply:
Thanks for your review, This is the code after I use the specified code style format. Will the last author not follow the format?

Reply:
Thanks for your review, I have fixed it. :D